.github/workflows/pull-request.yml

```diff
@@ -23,6 +23,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Setup redocly
+        run: npm i -g @redocly/cli@1.0.0-beta.124
+
+      - name: Bundle DAG api docs
+        working-directory: dag/node/
+        run: |
+          redocly bundle l0-public-v2.yml -o l0-public-v2.yml
+          redocly bundle l1-public-v2.yml -o l1-public-v2.yml
+          redocly bundle l0-cli-v2.yml -o l0-cli-v2.yml
```

Comment:
Isn't `cli` the same for L0 and L1? That's why I extracted CLI to upper level for currency.